### DISCUSSION TITLE
Orientdb enterprese-agent, unable to build the jar

### DISCUSSION THREAD

**Discussion by nandotorterolo** [orientechnologies/orientdb#10047] [Q&A] [answered]
Hi, I am trying to build the agent but test phase is not working, dev brach.
More details here: https://github.com/SAP/orientdb-enterprise-agent/issues/12
Do you know if jars are available to download so I can skip build it from source?
Thanks

**Comment by tglman** [comment, marked as answer]
Hi,

for releases you can find them here: https://repo1.maven.org/maven2/com/orientechnologies/agent/  make sure you take the same agent jar version of the distribution.

Regards

**Reply by nandotorterolo** [reply]
Many thanks @tglman appreciate your support!
